src/importSelect/ImportSelect.tsx

```diff
@@ -0,0 +1,189 @@
+import { useHeaderHeight } from '@react-navigation/elements'
+import { NativeStackScreenProps } from '@react-navigation/native-stack'
+import React, { useLayoutEffect } from 'react'
+import { useTranslation } from 'react-i18next'
+import { ScrollView, StyleSheet, Text, View } from 'react-native'
+import { SafeAreaView } from 'react-native-safe-area-context'
+import { useDispatch } from 'react-redux'
+import { cancelCreateOrRestoreAccount } from 'src/account/actions'
+import { OnboardingEvents } from 'src/analytics/Events'
+import ValoraAnalytics from 'src/analytics/ValoraAnalytics'
+import Card from 'src/components/Card'
+import Touchable from 'src/components/Touchable'
+import CloudCheck from 'src/icons/CloudCheck'
+import Lock from 'src/icons/Lock'
+import { nuxNavigationOptions } from 'src/navigator/Headers'
+import { navigateClearingStack } from 'src/navigator/NavigationService'
+import { Screens } from 'src/navigator/Screens'
+import { StackParamList } from 'src/navigator/types'
+import TopBarTextButtonOnboarding from 'src/onboarding/TopBarTextButtonOnboarding'
+import { goToNextOnboardingScreen, onboardingPropsSelector } from 'src/onboarding/steps'
+import useTypedSelector from 'src/redux/useSelector'
+import colors from 'src/styles/colors'
+import { typeScale } from 'src/styles/fonts'
+import { Shadow, Spacing } from 'src/styles/styles'
+
+export enum ChosenRestoreType {
+  Cloud = 'Cloud',
+  Mnemonic = 'Mnemonic',
+}
+
+type Props = NativeStackScreenProps<StackParamList, Screens.ImportSelect>
+
+function ActionCard({
+  title,
+  description,
+  icon,
+  onPress,
+  testID,
+}: {
+  title: string
+  description: string
+  icon: React.ReactNode
+  onPress?: () => void
+  testID?: string
+}) {
+  return (
+    <Card style={styles.card} rounded={true} shadow={Shadow.SoftLight} testID={testID}>
+      <Touchable borderRadius={8} style={styles.touchable} onPress={onPress}>
+        <View style={styles.cardContent}>
+          <View style={styles.topLine}>
+            {icon}
+            <Text style={styles.cardTitle}>{title}</Text>
+          </View>
+          <Text style={styles.cardDescription}>{description}</Text>
+        </View>
+      </Touchable>
+    </Card>
+  )
+}
+
+export default function ImportSelect({ navigation }: Props) {
+  const dispatch = useDispatch()
+  const headerHeight = useHeaderHeight()
+  const { t } = useTranslation()
+  const onboardingProps = useTypedSelector(onboardingPropsSelector)
+
+  const handleNavigateBack = () => {
+    dispatch(cancelCreateOrRestoreAccount())
+    ValoraAnalytics.track(OnboardingEvents.restore_account_cancel)
+    navigateClearingStack(Screens.Welcome)
+  }
+
+  useLayoutEffect(() => {
```

Comment:
If we want a header on this screen later we can use the changes that were removed in 3b181ea552784ee0f4244c760167311037cefe20.